photorepoapp-api/src/main/java/api/photorepoapp/photorepoappapi/controller/ImageController.java

```diff
@@ -0,0 +1,98 @@
+package api.photorepoapp.photorepoappapi.controller;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+
+
+import java.net.URI;
+
+import api.photorepoapp.photorepoappapi.filestore.AmazonStore;
+import api.photorepoapp.photorepoappapi.filestore.ImageRepo;
+import api.photorepoapp.photorepoappapi.models.Image;
+import imageBucket.Bucket;
+
+
+
+
+@RestController
+@RequestMapping
```

Comment:
Set our base API Url to be something along the lines of:

("/api/v1/[controller]")

Whenever we set up a new controller -- this helps organize things for the frontend when we want to make calls to the API